### ISSUE TITLE
Porting PassProtect to Firefox

### ISSUE THREAD

**Issue by caitmuenster** [OktaSecurityLabs/passprotect-js#1]
Hi there! It would be awesome to have this extension available to Firefox users. It should be fairly easy to port the Chrome version to Firefox. (There's a [guide for porting Chrome extensions to Firefox](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Porting_a_Google_Chrome_extension) on MDN.)

I'm the Add-ons Community Manager for Firefox and would be happy to help out with this if I can! Thanks so much.

**Comment by rdegges** [comment]
I've already got it in progress! I ran into a few issues I haven't been able to resolve yet -- any chance I could get your email? Would love to figure it out =)

**Comment by caitmuenster** [comment]
Absolutely! My email is cneiman [at] mozilla.com. You might also want to reach out to our developer mailing list at dev-addons [at] mozilla.org -- they'd be happy to help troubleshoot. :)

**Comment by andymarch** [comment]
I wrote a port of this for Firefox. https://github.com/andymarch/passprotect-firefox
At the moment it's not signed for AMO

**Comment by pc-mrousseau** [comment]
This would be amazing, has anyone made any progress on this in the last few years? @andymarch's looks great but having a signed one would be cool.
